### PR TITLE
Dev2021 shrav

### PR DESCRIPTION
**Pull request by shravanisamala**
Changes: 
- Added prerequisite page
- Changed navigation (Go buttons)
- Changed code so it would follow the style guide more
- Added reference page

### CODE REVIEW

**Review by adpoleski1 — DISMISSED** [review]
Everything looks good to me!  I reviewed the code and also ran through the app a few times.  Nice work!

**Comment by Jiayue-He** [comment]
Good job, Shravani. I've reviewed your code and it is good. I noticed is the width of the title, would it be 250 instead of 300 according to our study guide? And also the correct and incorrect mark, do we need to follow the study guide?

File: app.R [hunk 1]
```diff
@@ -10,239 +10,472 @@ MAX_TRIES <- 4
...
 hint <- c()
 correct_answer <- c()
 bank <- read.csv("distributionG.csv")
 bank <- data.frame(lapply(bank, as.character), stringsAsFactors = FALSE)
 
-ui <- dashboardPage(
-  skin = "blue",
-  dashboardHeader(
-    title = "Matching Distributions",
-    titleWidth = 300,
```
Comment (Jiayue-He) [inline app.R:23]: would the width of the title be 250?

**Comment by Jiayue-He** [comment]
Hello Shravani, I also noticed that LICENSE.md and README.md should be placed in a file named docs.

**Review by adpoleski1 — DISMISSED** [review]

**Comment by neilhatfield** [comment]
@Jiayue-He If you don't have any further changes for @shravanisamala to make, please click the Add Review button (located towards the top of the page and is green) and select the Approve option.

**Comment by neilhatfield** [comment]
@shravanisamala Please pull the result of my merge (https://github.com/EducationShinyAppTeam/Matching_Distributions/pull/11/commits/77cbc87d89f3eca58474331506cd641d5ae1bf87) down to your computer. You'll want to double check that 1) I didn't accidentally break the app and 2) fix any issues that I might have introduced.

**Review by Jiayue-He — DISMISSED** [review]

**Comment by shravanisamala** [comment]
@neilhatfield Hi Dr. Hatfield why does it keep saying I dismissed @adpoleski1 's review?

**Comment by neilhatfield** [comment]
> Hi Dr. Hatfield why does it keep saying I dismissed @adpoleski1 's review?

On protected branches (like this) a review gets "dismissed" when you push updates to the PR. In essence, it is a way to ensure that the approving reviews are for the actual code that will be used in the PR and not something else. By way of an extreme example, supposed that you accidentally swapped out the Matching Distributions code for the Prob. Application code. We'd want both Jiayue and Adam's reviews to be dismissed so that we wouldn't publish the wrong code.

**Pull request merged**